### ISSUE TITLE
Warning com.onesignal.PushRegistratorFCM->getApiKey

### ISSUE THREAD

**Issue by MayoudP** [OneSignal/react-native-onesignal#960]
**Description:**

Hello, currently having the same warning mentioned in the issue below. Is an update planned in the next coming days to fixed it for react-native ? 
This is a warning, I'm getting when trying to submit a new version for the Android Store...
https://github.com/OneSignal/OneSignal-Unity-SDK/issues/278#issue-575259072

**Environment**

 ```
  "react": "16.8.6",
    "react-native": "0.60.5",
    "react-native-onesignal": "^3.6.5",
```

**Steps to Reproduce Issue:**

1. Try to submit update on android console application. Getting a warning. 
<img width="216" alt="Screenshot 2020-03-24 at 14 56 53" src="https://user-images.githubusercontent.com/48682041/77433493-baf56380-6ddf-11ea-9d6f-7b485c79688d.png">

**Comment by Jacklol** [comment]
same problem

**Comment by yaronlevi** [comment]
This issue has been fixed in the [Android SDK](https://github.com/OneSignal/OneSignal-Android-SDK/issues/966). It will be great if we can get an update to the RN SDK to include this update.

**Comment by quynhapl** [comment]
> This issue has been fixed in the [Android SDK](https://github.com/OneSignal/OneSignal-Android-SDK/issues/966). It will be great if we can get an update to the RN SDK to include this update.

I am waiting for Flutter SDK, too!

**Comment by maxencehenneron** [comment]
Looks like we need to bump the one signal SDK from 3.12.7 to 3.13.0. I wonder if the upgrade is straight forward.

Looking at the release notes

```
Set and remove external user id now have completion callback
Callback for completion added to verify the status of the set or remove of an external user id (push & email)
```

we may have to refactor setExternalUserId to fix this issue.

**Comment by achmadfaturrizky** [comment]
I have the same problem, I hope it will be fixed immediately

**Comment by ermankuruoglu** [comment]
I also have same problem

**Comment by jkasten2** [comment]
@maxencehenneron The external user id callbacks are optional.

If you need to bump the OneSignal-Android-SDK native update version now edit this update it in [react-native-onesignal's `android/build.gradle` file](https://github.com/OneSignal/react-native-onesignal/blob/master/android/build.gradle#L27). Using `3.13.0` will give you the version where this warning is fixed.

However this warning is a false positive, you can safely release an update to your app without updating.

We are looking to release a `react-native-onesignal` update sometime next week. It will include the the set external user id callback which is why we haven't simply released an update with just the version bump.

**Comment by mrkindy** [comment]
Hello @jkasten2, we are waiting your release

**Comment by jkasten2** [comment]
This fixed is now included in the 3.7.1 release.
https://github.com/OneSignal/react-native-onesignal/releases
